src/openrct2/world/Map.cpp

```diff
@@ -129,11 +127,27 @@ void UnstashMap()
 {
     _tileIndex = std::move(_tileIndexStash);
     _tileElements = std::move(_tileElementsStash);
-    gMapSize = _mapSizeStash;
+    GetGameState().MapSize = _mapSizeStash;
     gCurrentRotation = _currentRotationStash;
     _tileElementsInUse = _tileElementsInUseStash;
 }
 
+CoordsXY GetMapSizeUnits()
```

Comment:
Had to move from an inline function as it references the static gameState variable.

Reply:
That's fine, not sure why it was done in the header anyway.